src/odemis/odemisd/start.py

```diff
@@ -298,7 +298,7 @@ def start_backend(self, modelfile):
                            "Try starting Odemis again, and use the \"Report a "
                            "problem\" function if the issue continues.\n\n"
                            "The full message is the following:\n %s" %
-                           (p.returncode, err[-1], "".join(err)))
+                           (p.returncode, err[-1], b"".join(err)))
```

Comment:
If err is a list of bytes, that is not what we expect. We want to show the message, so it should be *strings*. As it is, it will look strange (as the bytes are shown as `b'...'`). I'd suggest to `decode("utf-8", "ignore_error")` on out and err. It *could* be directly inside `proc_communicate()`.

Reply:
updated